### PR TITLE
Add validator version in validation metadata

### PR DESCRIPTION
**Pull request by fchabouis**
I add the validator version (found in cargo.tml) in the validation output. This is going to be useful when using it on transport.data.gouv.fr, to know which validations should be run again after something has been added to the validator.

### CODE REVIEW

File: src/metadatas.rs [hunk 1]
```diff
@@ -134,6 +136,7 @@ pub fn extract_metadata(gtfs: &gtfs_structures::RawGtfs) -> Metadata {
             .unwrap_or(&vec![])
             .iter()
             .any(|st| has_on_demand_pickup_dropoff(st, PickupDropOffType::CoordinateWithDriver)),
+        validator_version: validator_version.to_owned(),
```
Comment (AntoineAugusti) [inline src/metadatas.rs:139]: Googled `to_owned` vs `clone` and gosh it is difficult

File: readme.md [hunk 2]
```diff
@@ -45,7 +45,7 @@ Give useful information about the validated file content:
 | lines_with_custom_color_count | integer | Numer of routes found in `routes.txt` with a custom `route_color` or a custom `route_text_color`. Custom means different from the default values. |
 | some_stops_need_phone_agency | boolean | Some stops have a `continuous_pickup` or a `continuous_drop_off` field equal to `2`. |
 | some_stops_need_phone_driver | boolean | Some stops have a `continuous_pickup` or a `continuous_drop_off` field equal to `3`. |
-
+| validator_version | string | The validator version as found in cargo.tml file |
```
Comment (AntoineAugusti) [inline readme.md]:
```suggestion
| validator_version | string | The validator version as found in the `Cargo.toml` |
```
Fixing the filename

**Review by AntoineAugusti — APPROVED** [review]

File: src/metadatas.rs (hunk 1 above)
Comment (Tristramg) [reply src/metadatas.rs:139]: Indeed, this is one of the dark-ish corners. Just stick to one style ;)

**Review by Tristramg — APPROVED** [review]

File: src/metadatas.rs [hunk 3]
```diff
@@ -46,6 +47,7 @@ pub fn extract_metadata(gtfs: &gtfs_structures::RawGtfs) -> Metadata {
         .minmax()
         .into_option();
     let format = |d: chrono::NaiveDate| d.format("%Y-%m-%d").to_string();
+    let validator_version = env!("CARGO_PKG_VERSION");
```
Comment (antoine-de) [inline src/metadatas.rs:50]: this is never updated is it (or it has been only twice :p)?

There are also trick to get the `git describe` (tag+sha1) to always get a precise version if you're interested.

Comment (fchabouis) [reply src/metadatas.rs:50]: I intended to bump the version from now on, every time a change is made. What is the trick you are talking about?

**Pull request merged**